### PR TITLE
feat(KAN-16) : init commit #2

### PR DESCRIPTION
**Pull request by Razzaq10md**
We have created the Events Page API which displays Event details.
It includes:
1. Controllers
2. Program.cs
3. Appsettings.json

### CODE REVIEW

**Review by Sohailsd — APPROVED** [review]

File: .vs/slnx.sqlite
Comment (AbdulHannan99) [inline .vs/slnx.sqlite:1]: we dont need this file, Please work on the gitIgnore

File: .vs/tbb.events.api/FileContentIndex/304b4b94-045f-4c7a-b25f-c64ff071dbcc.vsidx
Comment (AbdulHannan99) [inline .vs/tbb.events.api/FileContentIndex/304b4b94-045f-4c7a-b25f-c64ff071dbcc.vsidx:1]: we dont need this file, Please work on the gitIgnore

File: .vs/tbb.events.api/v17/.wsuo
Comment (AbdulHannan99) [inline .vs/tbb.events.api/v17/.wsuo:1]: we dont need this file, Please work on the gitIgnore

File: .vs/tbb.events.api/v17/DocumentLayout.json
Comment (AbdulHannan99) [inline .vs/tbb.events.api/v17/DocumentLayout.json:1]: we dont need this file, Please work on the gitIgnore

File: tbb.events.api/.gitignore
Comment (AbdulHannan99) [inline tbb.events.api/.gitignore:1]: We have a gitignore but somehow its not working,Please work on the gitIgnore

File: tbb.events.api/tbb.events.api/appsettings.json
Comment (AbdulHannan99) [inline tbb.events.api/tbb.events.api/appsettings.json:1]: Remember to remove your connection strings

"DefaultConnection": "Server=LAPTOP-H64FNGON\\SQLEXPRESS;Database=EventsDatabase;Trusted_Connection=True;TrustServerCertificate=True"

**Review by AbdulHannan99 — CHANGES_REQUESTED** [review]
Please check the requested changes

**Pull request closed without merging**